apps/src/lib/kits/maker/PlaygroundConstants.js

```diff
@@ -30,7 +30,8 @@ if (experiments.isEnabled('maker-captouch')) {
 
 export const BOARD_EVENT_ALIASES = {
   // codeStudioName: 'playground-io-name'
-  singleTap: 'tap:single',
+  shake: 'tap:single',
+  singleTap:'tap:single',
```

Comment:
:+1: The other reason not to remove these is so that existing projects will continue to work.